recipes/piou/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "piou" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Andarius/piou/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 872eea7972f7fbf045bc5d80022d3e1fbf2214d140e9ba984470c9bf96ae7415
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - pip
+    - poetry
+  run:
+    - python >=3.9,<4.0
+    - rich >=10.11.0
+
+test:
```

Could you please add an import test to validate the package as well?
```suggestion
test:
  imports:
    - piou  
```

---

@dopplershift sorry for the delay, I pushed the modifications